src/content/docs/en/reference/adapter-reference.mdx

```diff
@@ -394,3 +394,51 @@ function createEdgeMiddleware(middlewareEntryPoint) {
     // emit a new physical file using your bundler
 }
 ```
+
+### envGetSecret
+
+To support [`astro:env` secrets](/en/guides/environment-variables/#getsecret), you need to update the `envGetSecret` adapter feature. Additionally, the `astro/env/setup` module allows you to provide an implementation for `getSecret()`. In your server entrypoint, call `setGetEnv()` as soon as possible:
```

Comment:
Can we have a line explaining what this is before jumping into using it? Maybe matching the other entries (functionPerRoute was there before it was deleted in this branch. Maybe something like:

(I don't know if this should be `true`, or `"stable"` or what.)

``````suggestion
This is a feature to allow your adapter to retrieve secrets configured by users in `env.schema`.

Enable the feature by passing `true` to the adapter:

```js title="my-adapter.mjs" ins={9-11}
export default function createIntegration() {
  return {
    name: '@matthewp/my-adapter',
    hooks: {
      'astro:config:done': ({ setAdapter }) => {
        setAdapter({
          name: '@matthewp/my-adapter',
          serverEntrypoint: '@matthewp/my-adapter/server.js',
          adapterFeatures: {
              envGetSectret: true
          } 
        });
      },
    },
  };
}
```

The `astro/env/setup` module allows you to provide an implementation for `getSecret()`. In your server entrypoint, call `setGetEnv()` as soon as possible:
``````